core/src/App.html

```diff
@@ -220,6 +218,10 @@
       cursor: pointer;
     }
 
+    .fd-page{
```

Comment:
I agree this is the right way to do it, otherwise it will be inconsistent, and we will continue having displaying issues. But I am not convinced with the fact that we are providing a feature but it does not work out of the box, since every app using luigi needs to wrap the iframe in a container styled with this z-index value.

Reply:
This z-index value is assigned at Lugii Core's level so it works out of the box. The app developer doesn't have to implement any z-index value to the container.